southpark-search/tests/distributed/README.md

```diff
@@ -0,0 +1,669 @@
+## Terraform for setting up cloud infrastructure
+
+Configure AWS provider for Terraform as per README:
+https://github.com/jina-ai/cloud-ops
+
+The following infrastructure is setup using Terraform:
+
+We fetch the ip adresses of the pod-machines in the cloud VPN, putting them into JINA_ENCODER_HOST and JINA_INDEX_HOST environment variables
+Send the curl command below to the jinad on the flow machine in order to spin up the flow:
+```
+curl -s --request PUT "http://localhost:8000/v1/flow/yaml" \
+    -H  "accept: application/json" \
+    -H  "Content-Type: multipart/form-data" \
+    -F "uses_files=@pods/encode.yml" \
+    -F "uses_files=@pods/extract.yml" \
+    -F "uses_files=@pods/index.yml" \
+    -F "pymodules_files=@pods/text_loader.py" \
+    -F "yamlspec=@tests/distributed/flow-query.yml"
+```
+
+Docker image generated via `tests/distributed/Dockerfile` jinad will automatically be running on each machine brought up by Terraform.
+
+The provisioning would hence be:
+- Set up `Index` instance, make it use the Dockerfile
+- Set up `Encode` instance, make it use the Dockerfile
+- On the `Flow` instance, run the curl command. Extract the IPs of Index and Encode, make them available to Env variables
+
+The terraform script here spins up AWS resources, this is the output of running
+`terraform apply`:
+
+```
+data.aws_iam_policy_document.assume_role_policy: Refreshing state...
+
+An execution plan has been generated and is shown below.
+Resource actions are indicated with the following symbols:
+  + create
+ <= read (data resources)
+
+Terraform will perform the following actions:
+
+  # data.aws_subnet_ids.default will be read during apply
+  # (config refers to values not yet known)
+ <= data "aws_subnet_ids" "default"  {
+      + id     = (known after apply)
+      + ids    = (known after apply)
+      + tags   = (known after apply)
+      + vpc_id = (known after apply)
+    }
+
+  # aws_alb.application_load_balancer will be created
+  + resource "aws_alb" "application_load_balancer" {
+      + arn                        = (known after apply)
```

Comment:
Put this within the following elements to avoid the overwhelming code block
```
<details> 
<summary>Click here to see the outputs </summary>

your codes

</details>
```

Reply:
Yes sure, updating the PR